workitem/link/link_repository.go

```diff
@@ -307,3 +311,36 @@ func (r *GormWorkItemLinkRepository) Save(ctx context.Context, lt app.WorkItemLi
 	result := ConvertLinkFromModel(res)
 	return &result, nil
 }
+
+// ListWorkItemChildren get all child work items
+func (r *GormWorkItemLinkRepository) ListWorkItemChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
+	wilt := WorkItemLinkType{}
+	r.db.Where("forward_name = ?", "parent of").First(&wilt)
+
+	where := "id in (select target_id from work_item_links where source_id = ? and link_type_id = ?)"
```

Comment:
Use `%s` and replace with `workitem.WorkItemLink{}.TableName()` but don't use `work_item_links` directly.